samples/client/petstore/powershell_test/src/IO.Swagger/API/PetApi.ps1

```diff
@@ -0,0 +1,164 @@
+        $PSBoundParameters | Out-DebugParameter | Write-Debug
+
+        $Script:PetApi.FindPetsByStatus(
+            ${status}
+        )
+    }
+}
+
+function Invoke-PetApiFindPetsByTags {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 1, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $true)]
+        [[String]]
+        ${tags}
+    )
+
+    Process {
+        'Calling method: PetApi-FindPetsByTags' | Write-Verbose
+        $PSBoundParameters | Out-DebugParameter | Write-Debug
+
+        $Script:PetApi.FindPetsByTags(
+            ${tags}
+        )
+    }
+}
+
+function Invoke-PetApiGetPetById {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 1, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $true)]
+        [Int64]
+        ${petId}
+    )
+
+    Process {
+        'Calling method: PetApi-GetPetById' | Write-Verbose
+        $PSBoundParameters | Out-DebugParameter | Write-Debug
+
+        $Script:PetApi.GetPetById(
+            ${petId}
+        )
+    }
+}
+
+function Invoke-PetApiUpdatePet {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 1, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $true)]
+        [IO.Swagger.Model.Pet]
+        ${body}
+    )
+
+    Process {
+        'Calling method: PetApi-UpdatePet' | Write-Verbose
+        $PSBoundParameters | Out-DebugParameter | Write-Debug
+
+        $Script:PetApi.UpdatePet(
+            ${body}
+        )
+    }
+}
+
+function Invoke-PetApiUpdatePetWithForm {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 1, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $true)]
+        [Int64]
+        ${petId},
+        [Parameter(Position = 2, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $false)]
+        [String]
+        ${name},
+        [Parameter(Position = 3, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $false)]
+        [String]
+        ${status}
+    )
+
+    Process {
+        'Calling method: PetApi-UpdatePetWithForm' | Write-Verbose
+        $PSBoundParameters | Out-DebugParameter | Write-Debug
+
+        $Script:PetApi.UpdatePetWithForm(
+            ${petId},
+            ${name},
+            ${status}
+        )
+    }
+}
+
+function Invoke-PetApiUploadFile {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 1, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $true)]
+        [Int64]
+        ${petId},
+        [Parameter(Position = 2, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $false)]
+        [String]
+        ${additionalMetadata},
+        [Parameter(Position = 3, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true, Mandatory = $false)]
+        [IO.Swagger.Model.String]
```

Comment:
Found a fix and pushed the commit https://github.com/swagger-api/swagger-codegen/pull/5789/commits/dc31206a7afb0146187163f92f5ea66f21150e17